baselines/dom.generated.d.ts

```diff
@@ -14852,6 +14853,59 @@ interface EventSourceInit {
     readonly withCredentials: boolean;
 }
 
+interface AnimationOptions {
+    id?: DOMString;
```

Comment:
we so not use this type else where. just use `string`.

Reply:
ok => commit will be amended accordingly